### PR TITLE
year will now update to current year

### PR DESCRIPTION
**Pull request by sydbails**
added variable for date and year. lmk if this is what you had in mind. - syd

### CODE REVIEW

File: src/components/modules/Footer/Footer.js [hunk 1]
```diff
@@ -37,7 +39,7 @@ export default class Footer extends React.Component {
           </Link>
         </div>
         <div className={style.OrgInfoArea}>
-          Terms of Service | Privacy Policy | ©Dev Launchers, 2020.
+          Terms of Service | Privacy Policy | ©Dev Launchers, {year}.
```
Comment (third774) [inline src/components/modules/Footer/Footer.js]: One nit that is _not_ important — I would just inline this:

```js
           Terms of Service | Privacy Policy | ©Dev Launchers, {new Date().getFullYear()}. 
```

Comment (sydbails) [reply src/components/modules/Footer/Footer.js]: I can change it - lmk.

**Comment by pyxld-kris** [comment]
Looks good, merging into master!

**Pull request merged**